examples/example_precompiles_vm/example_precompiles_vm.cpp

```diff
@@ -52,8 +52,8 @@ static evmc_result execute(evmc_vm* /*unused*/,
                            evmc_host_context* /*unused*/,
                            enum evmc_revision rev,
                            const evmc_message* msg,
-                           const uint8_t*,
-                           size_t)
+                           const uint8_t* /*unused*/,
```

Comment:
Is this fine or too verbose?

Reply:
I think too verbose. I see some value in commenting what is exactly unused, like `/*code*/` (probably not possible with clang-tidy), but `unused` doesn't add any new info.

Reply:
Apparently, you can add any comment, even `/**/`.

Reply:
cool

Reply:
Ah it just allows these manually typed comments... For some reason I thought it can insert them automatically

Reply:
No. It will insert `/*unused*/` as a fix.